ui/main/src/app/components/activityarea/view/ActivityareaView.spec.ts

```diff
@@ -261,7 +261,7 @@ describe('ActivityAreaView', () => {
         const activityAreaPage = await firstValueFrom(activityAreaView.getActivityAreaPage());
         expect(activityAreaPage.activityAreaClusters[0].lines).toHaveSize(2);
         expect(activityAreaPage.activityAreaClusters[0].lines[0].entityId).toEqual('ENTITY1');
-        expect(activityAreaPage.activityAreaClusters[0].lines[0].connectedUsers).toEqual(['John Smith', 'currentUser']);
+        expect(activityAreaPage.activityAreaClusters[0].lines[0].connectedUsers).toEqual(['John Smith', 'James']);
```

Comment:
I suggest 
+ to rewrite  test description which is not relevant anymore
+ add a third connected user with no first name / no last name